JWBestPracticeApps/Podfile

```diff
@@ -5,6 +5,8 @@
 
 workspace 'JWBestPracticeApps'
 
+$PRIVATE_SOURCE = 'https://bitbucket.org/vualtomobile/vudrmfairplay-pods.git'
```

Comment:
Overall looks good. QQ here, the SDK source is to use the pod spec served in the bitbucket repo right? And there is no issue with this being public facing, since this repo is public.